CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.0.1]
+
 ### Fixed
 Fix e2e tests in the contract template - [#1537](https://github.com/paritytech/cargo-contract/pull/1537)
```

Comment:
```suggestion
- Fix e2e tests in the contract template - [#1537](https://github.com/paritytech/cargo-contract/pull/1537)
```

Let's adhere to the https://keepachangelog.com formatting in case someone parses it.